playbooks/preflight-check.yml

```diff
@@ -0,0 +1,64 @@
+---
+# Copyright 2017, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+- name: Preflight check
+  hosts: "all_containers"
```

Comment:
Might want to have this ping all hosts too. `hosts:all_containers`

Reply:
Did you mean hosts:all_hosts?

Reply:
no. the line would read

``` yaml
- name: Preflight check
  hosts: "hosts:all_containers"
  ...
```